examples/linq/2.qpu_connection.ipynb

```diff
@@ -0,0 +1,790 @@
+    "\n",
+    "This notebook should be relevant to Tangelo users that prefer to install all the dependencies on their machine, use their own credentials, and be billed directly by the target quantum cloud service(s).\n",
+    "\n",
+    "This approach is an alternative to using [QEMIST Cloud](https://goodchemistry.com/qemist-cloud/), which we intend to make a single and convenient entry point to reach many different platforms. What's pretty cool with this other option is that users do not have to set up the environment required by the quantum cloud(s) of their choice (e.g Braket, Azure Quantum, etc) or even an account with those services. A QEMIST Cloud account provides a single entry point enabling you to reach all of them, and pay with your QEMIST Cloud credits.\n",
+    "\n",
+    "\n",
+    "## Table of contents\n",
+    "* [1. IonQ Rest API](#1)\n",
+    "    * [1.1 References](#1.1)\n",
+    "    * [1.2 Environment](#1.2)\n",
+    "    * [1.3 Quantum Circuit](#1.3)\n",
+    "    * [1.4 IonQConnection class](#1.4)\n",
+    "* [2. Azure Quantum](#2)\n",
+    "* [3. Braket](#3)\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "- Tangelo needs to be installed in your environment.\n",
+    "- Ensure you have valid ID tokens and logins for the services of your choice (IonQ, Braket...), and that your environment is ready to use them (e.g install their SDK and dependencies according to the instructions they provide)\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. IonQ Rest API <a class=\"anchor\" id=\"1\"></a>\n",
+    "\n",
+    "\n",
+    "### 1.1 References <a class=\"anchor\" id=\"1.1\"></a>\n",
+    "\n",
+    "Tangelo provides a few Python wrappers around the REST API provided by IonQ, to facilitate job submission and monitoring. The most up-to-date list of supported features and information regarding the IonQ API is available at https://docs.ionq.co\n",
+    "\n",
+    "### 1.2 Environment <a class=\"anchor\" id=\"1.2\"></a>\n",
+    "\n",
+    "Users allowed to use the IonQ API should have an ID token, e.g a string of alphanumeric characters and dashes, which can be obtained through the [IonQ dashboard](https://cloud.ionq.com/). Users need to set their `IONQ_APIKEY` environment variable to this value; here are two ways to do it:\n",
+    "\n",
+    "- in your terminal (`export IONQ_APIKEY=<value>`)\n",
+    "- or, in your Python script (`os.environ['IONQ_APIKEY'] = <value>`)\n",
+    "\n",
+    "Here's an example of what it could look like for you (make sure you use a valid key, or you'll get an \"unauthorized\" error :) )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"IONQ_APIKEY\"] = '2T14z1YQEzMLCwuYM110oXPDT2h850E4'"
```

Comment:
Is it a valid key? Should we leave it there?

Reply:
It's not a valid key :)